tldw_Server_API/tests/e2e/test_data.py

```
@@ -44,7 +44,7 @@ def sample_text_content(seed: Optional[str] = None) -> str:
 
         # Create unique content based on seed or timestamp
         unique_id = seed or datetime.now().isoformat()
-        hash_suffix = hashlib.md5(unique_id.encode()).hexdigest()[:8]
+        hash_suffix = hashlib.md5(unique_id.encode()).hexdigest()[:8]  # nosec B324
```

Ruff (0.15.1)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)